### DISCUSSION TITLE
What do these buttons do?

### DISCUSSION THREAD

**Discussion by givihuda** [dbeaver/dbeaver#16509] [Q&A]
![image](https://user-images.githubusercontent.com/74785256/168840768-679eaaab-fa61-4a58-8ac4-732624cf2e0f.png)

There's no hover tooltip.

They're in the query results pane, after the refresh button.

**Comment by LonwoLonwo** [comment]
Hello @givihuda 

This is forward and backward - history navigation buttons - https://github.com/dbeaver/dbeaver/wiki/Data-Editor

![9JDR65BMHb](https://user-images.githubusercontent.com/45152336/168846617-220710b7-00fc-4498-9ee5-708d0e784fea.gif)

**Reply by givihuda** [reply]
Why not have a hover tooltip for them?

**Reply by LonwoLonwo** [reply]
But they have tooltips, you can see them in the video above.

![2022-07-12 13_01_17-What do these buttons do_ · Discussion #16509 · dbeaver_dbeaver — Mozilla Firefo](https://user-images.githubusercontent.com/45152336/178465453-8d6603b7-861e-46dd-a4d1-e493b7d85a74.png)

**Reply by givihuda** [reply]
I'm not able to reproduce what the video shows... my green left-arrow at the moment on that view is grayed-out anyway... regardless, hovering should show _something_ regardless of context, such as "this is what this thing is for". 

![image](https://user-images.githubusercontent.com/74785256/180067990-784055d2-a349-456b-827d-513221f24390.png)

I can't tell why the button's unclickable, nor what it's for, because hovering over it describes nothing.

The button just happening to show a SELECT query upon hover _implicitly_ describes its behavior based on the context that you clicked into (I suppose), but there's no indication otherwise apparently (even with you going through those steps in the video) of what the buttons are _for_ upon hover... and it appears you have to have already used the button just to get a hover tooltip describing what it's for 😆

No button should require you to click it before being able to hover over it to see what it's for... do you understand why? Who knows what a button will do until you click it...
